### ISSUE TITLE
get_info or tidyinfo is handling some data badly. Possibly incorrectly inserting anomalous data.

### ISSUE THREAD

**Issue by rynkwn** [anttsou/qmj#18]
Ex: In our financials data frame:
![capture](https://cloud.githubusercontent.com/assets/4651792/12108189/c409d4da-b342-11e5-8a09-a79aed4d705a.PNG)

However, looking directly at the quantmod data, we have:

CASHFLOWS:
![image](https://cloud.githubusercontent.com/assets/4651792/12108191/d18ff3f0-b342-11e5-990e-5fc457f7a9f6.png)

BALANCESHEETS:
![image](https://cloud.githubusercontent.com/assets/4651792/12108196/e6ca71d2-b342-11e5-965e-e9a247d827a7.png)

INCOME STATEMENTS:
![image](https://cloud.githubusercontent.com/assets/4651792/12108201/fe7d174e-b342-11e5-8522-132dc26cd2a2.png)

In other words, either get_info or tidyinfo is poorly handling its receipt of information to incorrectly store data. Will look into this further.

**Comment by rynkwn** [comment]
Some of its numbers, for example DP.DPL for Depreciation.Depletion, don't appear to come from the correct source. In fact, I'm unsure where it's getting those numbers from. Checking the adjoining rows doesn't make it clear that some off-by-one error is assigning LE's values elsewhere, but it's possible that it's incremented sufficiently by that point that the "off-by-one" error is pretty enormous. That, and logically then the last X rows should essentially be lacking data. Something that isn't true.

Will look into this more deeply tomorrow.

**Comment by rynkwn** [comment]
Playing with it now.

So taking a small sample of ~100 companies around LE still attributes incorrect data to LE.
However, the get_info function on LE alone does produce correct data.

If I take the last company in my subset, LPI, I find that the raw financials does indeed produce the correct cash flow statement.

Hm. Perhaps quantmod is grabbing the wrong data?

**Comment by rynkwn** [comment]
Scratch that, it seems as if the correct data is being grabbed by get_info. Unsure why I thought otherwise just above. Will check tidyinfo.

**Comment by rynkwn** [comment]
Tidyinfo is producing the correct data. Though it's possibly being somewhat harsh in cutting out 2012 data for LE due to the lack of a balance sheet, it doesn't seem pressing.

Will try re-generating our financials data set to see if inaccuracy persists. Maybe data set was produced by an older, buggy version?

**Comment by rynkwn** [comment]
Correction*

Upon closer inspection, my conclusion is that get_info/tidyinfo is correctly grabbing and producing data.
My guess is that, as we only have access to the past 4 10-K filings, we were grabbing weird/old data from LE.

To be completely sure, I've also re-built our financials data set. LE now has reasonable data. On top of that, it's also moved forward 

![image](https://cloud.githubusercontent.com/assets/4651792/12116956/610bb1f2-b38b-11e5-9088-85b989351df4.png)

It's also moved forward around 700 rows. Implicitly this means that there was an enormous gain in information in the preceding companies, which is also concerning.
